src/EventListener/InvalidateRequestSubscriber.php

```diff
@@ -26,9 +35,7 @@ public function __construct(CacheMiddleware $cacheMiddleware)
     public static function getSubscribedEvents() : array
     {
         return [
-            GuzzleBundleCacheEvents::INVALIDATE => [
-                ['invalidate', 0],
-            ],
+            GuzzleBundleCacheEvents::INVALIDATE => 'invalidate',
```

Comment:
why ? with the "long" syntax you can be more specific on the priority.

Reply:
agree with you, reverted